Allow full url linking

marvinody opened the pull request:
Can now post a full url link without shortening and it should be parsed.
short urls (ie: https://youtu.be/ZcJjMnHoIBI) will not work

marvinody committed:
Allow full url linking

Can now post a full url link without shortening and it should be parsed.
short urls (ie: https://youtu.be/ZcJjMnHoIBI) will not work

meew0 commented:
I initially didn't do this because I wanted to avoid embedding the videos upon queueing them, but meh, it doesn't hurt.

marvinody commented:
I just made a new channel in my server for solely requests and removed embedding. It didn't work because I think client permissions override channel perms. and everyone had admin on mine. Point is that they didn't want to waste the 5 seconds to get the code and I just modified it quickly
